keyboards/skyloong/gk75/q1/info.json

```diff
@@ -0,0 +1,820 @@
+
+            {"matrix": [3, 0], "x": 0, "y": 38, "flags": 4},
+            {"matrix": [3, 1], "x": 26, "y": 38, "flags": 4},
+            {"matrix": [3, 2], "x": 40, "y": 38, "flags": 4},
+            {"matrix": [3, 3], "x": 55, "y": 38, "flags": 4},
+            {"matrix": [3, 4], "x": 70, "y": 38, "flags": 4},
+            {"matrix": [3, 5], "x": 86, "y": 38, "flags": 4},
+            {"matrix": [3, 6], "x": 99, "y": 38, "flags": 4},
+            {"matrix": [3, 7], "x": 114, "y": 38, "flags": 4},
+            {"matrix": [3, 8], "x": 129, "y": 38, "flags": 4},
+            {"matrix": [3, 9], "x": 143, "y": 38, "flags": 4},
+            {"matrix": [3, 10], "x": 158, "y": 38, "flags": 4},
+            {"matrix": [3, 11], "x": 173, "y": 38, "flags": 4},
+            {"matrix": [4, 14], "x": 209, "y": 26, "flags": 4},
+            {"matrix": [3, 12], "x": 187, "y": 38, "flags": 4},
+            {"matrix": [3, 13], "x": 198, "y": 38, "flags": 4},
+            {"matrix": [3, 14], "x": 224, "y": 38, "flags": 4},
+
+            {"matrix": [4, 0], "x": 0, "y": 51, "flags": 4},
+            {"matrix": [0, 1], "x": 18, "y": 51, "flags": 4},
+            {"matrix": [4, 1], "x": 33, "y": 51, "flags": 4},
+            {"matrix": [4, 2], "x": 48, "y": 51, "flags": 4},
+            {"matrix": [4, 3], "x": 62, "y": 51, "flags": 4},
+            {"matrix": [4, 4], "x": 77, "y": 51, "flags": 4},
+            {"matrix": [5, 6], "x": 92, "y": 51, "flags": 4},
+            {"matrix": [4, 6], "x": 106, "y": 51, "flags": 4},
+            {"matrix": [4, 7], "x": 121, "y": 51, "flags": 4},
+            {"matrix": [4, 8], "x": 136, "y": 51, "flags": 4},
+            {"matrix": [4, 9], "x": 151, "y": 51, "flags": 4},
+            {"matrix": [4, 10], "x": 165, "y": 51, "flags": 4},
+            {"matrix": [4, 11], "x": 180, "y": 51, "flags": 4},
+            {"matrix": [4, 13], "x": 209, "y": 51, "flags": 4},
+
+            {"matrix": [5, 0], "x": 0, "y": 64, "flags": 4},
+            {"matrix": [5, 1], "x": 18.4, "y": 64, "flags": 4},
+            {"matrix": [5, 2], "x": 37, "y": 64, "flags": 4},
+            {"matrix": [5, 4], "x": 55, "y": 64, "flags": 4},
+            {"matrix": [5, 5], "x": 92, "y": 64, "flags": 4},
+            {"matrix": [4, 5], "x": 103, "y": 64, "flags": 4},
+            {"matrix": [5, 7], "x": 129, "y": 64, "flags": 4},
+            {"matrix": [5, 9], "x": 147, "y": 64, "flags": 4},
+            {"matrix": [5, 10], "x": 162, "y": 64, "flags": 4},
+            {"matrix": [5, 11], "x": 176, "y": 64, "flags": 4},
+            {"matrix": [5, 12], "x": 195, "y": 64, "flags": 4},
+            {"matrix": [5, 13], "x": 209, "y": 64, "flags": 4},
+            {"matrix": [5, 14], "x": 224, "y": 64, "flags": 4}
+        ]
+    },
+    "url": "https://github.com/JZ-Skyloong",
+    "usb": {
+        "force_nkro": true,
+        "vid": "0x1EA7",
+        "max_power": 380
+    },
+    "layouts": {
+        "LAYOUT_all": {
```

Comment:
If you don't actually intend to have a single PCB that supports both ANSI and ISO layouts, you should not combine layouts this way.  Instead, move the `LAYOUT_ansi_*` and `LAYOUT_iso_*` definitions to `info.json` files in the corresponding subfolders, and rename `LAYOUT_ansi_all` and `LAYOUT_iso_all` to `LAYOUT_all` in each of them.

Reply:
I see and updated them.